kbdlight.c

```diff
@@ -68,6 +69,10 @@ int main(int argc, char* argv[]) {
   } else if(!strcmp(argv[1], "get")) {
     printf("%d\n", current);
     return 0;
+  } else if(!strcmp(argv[1], "getp")) {
+    float percentage = round((float)current / 255 * 100.0);
+    printf("%.f\n", percentage);
```

Comment:
Do you really need the decimals here? Otherwise you can just use decimal, and use `current * 100 / 255` above to avoid it rounding down to zero.